lib/taskana-core/src/test/java/acceptance/task/CreateTaskAccTest.java

```diff
@@ -50,6 +51,11 @@
   private final ClassificationService classificationService =
       taskanaEngine.getClassificationService();
 
+  @BeforeEach
```

Comment:
maybe we should change this to an after each? or this can be resolved with a dirtyContext Annotation (currently in discussion at the cop)

Reply:
what about a try {} finally {} block for that one test?

Otherwise I think this is totally fine and acceptable